core/schema/git_test.go

```diff
@@ -0,0 +1,31 @@
+package schema
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMatchVersion(t *testing.T) {
+	vers := []string{"v1.0.0", "v1.0.1", "v2.0.0", "path/v1.0.1", "path/v2.0.1"}
+
+	match1, err := matchVersion(vers, "v1.0.1", "/")
+	require.NoError(t, err)
+	require.Equal(t, "v1.0.1", match1)
+
+	match2, err := matchVersion(vers, "v1.0.1", "path")
+	require.NoError(t, err)
+	require.Equal(t, "path/v1.0.1", match2)
```

Comment:
Worth testing `/path` here too, I'd expect that to be the actual value at runtime.

Reply:
Added! fwiw I don't think I had `/path` as the subpath, but it's handled either way now